lbrynet/lbrynet_daemon/Publisher.py

```diff
@@ -38,8 +39,11 @@ def __init__(self, session, lbry_file_manager, wallet):
     def start(self, name, file_path, bid, metadata):
         log.info('Starting publish for %s', name)
         def _show_result():
-            log.info("Success! Published %s --> lbry://%s txid: %s", self.file_name, self.publish_name, self.txid)
-            return defer.succeed(self.txid)
+            log.info("Success! Published %s --> lbry://%s txid: %s nout: %d", self.file_name, self.publish_name, self.txid, self.nout)
```

Comment:
nit: wrap the long line (I'll add a pylint check for this at some point... I just don't feel like cleaning up all of them...)